socorro/processor/mozilla_transform_rules.py

```diff
@@ -813,8 +813,8 @@ def __init__(self, config):
     def version(self):
         return '1.0'
 
-    def _get_version_data(self, product, version, release_channel, build_id):
-        key = '%s:%s:%s:%s' % (product, version, release_channel, build_id)
+    def _get_version_data(self, product, version, build_id):
+        key = '%s:%s:%s' % (product, version, build_id)
```

Comment:
I have added a docstring to that function, let me know if that explains it correctly.